services/webinar/authenticate.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import { Headers } from 'node-fetch';
+import querystring from 'querystring';
+
+import { fetchWithBody, apiBase } from './utils';
+
+function parseExpirationDate(expiresIn: string): number {
+  const secondsFromNow = parseInt(expiresIn, 10);
+  const millisFromNow = secondsFromNow * 1000;
+  const expirationDate = Date.now() + millisFromNow;
```

Comment:
We use moment in other parts of the app, it might help here. If not, we should have a test to cover this code.

Reply:
discussed - not worried about using moment as we also use date-fns. Do need the test though